pkg/anthropic/config.go

```diff
@@ -0,0 +1,9 @@
+package anthropic
+
+type Config struct {
+	APIKey string
+}
+
+func (c *Config) GetIngestrURI() string {
+	return "anthropic://?api_key=" + c.APIKey
```

Comment:
We should url escape the key. A reserved character might break parsing in unexpected ways.